docs/presentations.md

```diff
@@ -133,6 +133,10 @@ Presented Summer 2010 at [Yahoo](http://yahoo.com)
 # Apache Mesos - Training
 _(Listed in date descending order)_
 
+## Docker, Apache Mesos & DCOS: Run and manage cloud datacenter ([Video](https://www.packtpub.com/networking-and-servers/docker-apache-mesos-dcos-run-and-manage-cloud-datacenter-video))
+Manuj Aggarwal, TetraTutorials Team
```

Comment:
Let's remove the "TetraTutorials Team" designation, for consistency.

Reply:
I have created another PR without TetraTutorials Team